packages/modules/imodel-browser/src/containers/ITwinGrid/useITwinFavorites.ts

```diff
@@ -52,9 +52,11 @@ export const useITwinFavorites = (
             Accept: "application/vnd.bentley.itwin-platform.v1+json",
           },
         });
-        if (!result || result.status !== 200) {
+
+        if (!result || (result.status !== 200 && result.status !== 204)) {
```

Comment:
this api cant return a 204, https://developer.bentley.com/apis/itwins/operations/add-itwin-to-my-favorites/#response-ref